.eslintrc.js

```diff
@@ -1,3 +1,8 @@
+/* eslint-disable import/no-commonjs */
+
+const path = require( 'path' );
+const customResolverPath = path.join( __dirname, './.eslint-resolver.js' );
```

Comment:
Won't the result of this join be simply "./.eslint-resolver.js" which we could instead define as a string key of the `settings`, not a computed property?